### PR TITLE
Home edit: Fix model card settings drop-down not accessible

### PR DESCRIPTION
**Pull request by florian-h05**
Fixes issue mentioned in https://github.com/openhab/openhab-webui/pull/2144#issuecomment-1784114001.

This is not the best solution, but it works now.

The issue looked like this:
image

### CODE REVIEW

**Comment by florian-h05** [comment]
@ghys I was able to fix the described problem, but IMO the solution is not really "clean". WDYT?

File: bundles/org.openhab.ui/web/src/pages/settings/pages/home/home-edit.vue [hunk 1]
```diff
@@ -83,7 +82,9 @@
                 </f7-list-item>
               </f7-list>
             </div>
+          </f7-block>
 
+          <f7-block style="z-index: -1">
```
Comment (florian-h05) [inline bundles/org.openhab.ui/web/src/pages/settings/pages/home/home-edit.vue:87]: FYI I had to switch from col to block to make z-index work, and I had to wrap the bottom settings in their own block, which IMO makes sense anyway.

Comment (florian-h05) [inline bundles/org.openhab.ui/web/src/pages/settings/pages/home/home-edit.vue:87]: FYI I had to switch from col to block to make z-index work, and I had to wrap the bottom settings in their own block, which IMO makes sense anyway.

**Comment by florian-h05** [comment]
@ghys Sorry for pinging you again, can you please have a look at the few lines of this PR?

File: bundles/org.openhab.ui/web/src/pages/settings/pages/home/home-edit.vue [hunk 2]
```diff
@@ -32,21 +32,20 @@
...
-        <f7-block class="block-narrow" style="padding-bottom: 8rem" v-else-if="ready && modelReady && !previewMode">
-          <f7-col>
-            <f7-block-title>Page Configuration</f7-block-title>
-            <config-sheet
-              :parameterGroups="pageWidgetDefinition.props.parameterGroups || []"
-              :parameters="pageWidgetDefinition.props.parameters || []"
-              :configuration="page.config"
-              @updated="dirty = true" />
-          </f7-col>
+        <div v-else-if="!previewMode">
```
Comment (ghys) [inline bundles/org.openhab.ui/web/src/pages/settings/pages/home/home-edit.vue:36]: I was just wondering why you removed the 2 other conditions (`ready && modelReady`).
Oh well I suppose it's fine...

**Review by ghys — APPROVED** [review]
Thanks @florian-h05 - lgtm!

**Pull request merged**